CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.15.4 (2021-03-24)
+
+- Fixed the `skyThemeIf` and the `skyThemeClass` directives to work properly when `SkyThemeService` is not provided. [#217](https://github.com/blackbaud/skyux-theme/pull/217)
```

Comment:
```suggestion
- Fixed the `skyThemeIf` and `skyThemeClass` directives to work properly when `SkyThemeService` is not provided. [#217](https://github.com/blackbaud/skyux-theme/pull/217)
```
Since both code snippets modify the same word, we only need a single article for both of them.